sunflower-farmers/sunflower-farmers
# Change token symbol to $SFF

This PR will change how the token is referenced on the game! 

Instead of $10 use 10 $SFF.

To test:
* Check the welcome modal
* Check Total Supply modal
* Check Fruit Board
